samples/wifi/scan/Kconfig

```diff
@@ -60,6 +60,16 @@ config WIFI_SCAN_DWELL_TIME_PASSIVE
 	help
 	  Passive scan dwell time (in ms) per channel.
 
+config WIFI_SCAN_TYPE_ACTIVE
+	bool "Active scan"
+	help
+	  Enable this profile to do active scan where allowed.
+
+config WIFI_SCAN_TYPE_PASSIVE
+	bool "Passive scan"
+	help
+	  Enable this profile to do passive scan on all channels.
+
```

Comment:
choice?

Reply:
I had this as a choice, but cannot use select/imply with choice symbols, so, had them as normal symbols.